<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFETCSPCImporterLifetimeView</class>
 <widget class="QDialog" name="QFETCSPCImporterLifetimeView">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCSPC Lifetime Viewer</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
   <item>
    <widget class="QLabel" name="labDescription">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkLogX">
        <property name="text">
         <string>log X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkLogY">
        <property name="text">
         <string>log Y</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>667</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelectRange">
        <property name="text">
         <string>select lifetime range</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/lib/draw_rectangle.png</normaloff>:/lib/draw_rectangle.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFPlotter" name="plotter" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFPlotter</class>
   <extends>QWidget</extends>
   <header>qfplotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFETCSPCImporterLifetimeView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFETCSPCImporterLifetimeView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkLogX</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFETCSPCImporterLifetimeView</receiver>
   <slot>updatePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkLogY</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFETCSPCImporterLifetimeView</receiver>
   <slot>updatePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>helpRequested()</signal>
   <receiver>QFETCSPCImporterLifetimeView</receiver>
   <slot>showHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatePlot()</slot>
  <slot>showHelp()</slot>
 </slots>
</ui>
